source/framework/vx_user_data_object.c

```diff
@@ -155,13 +221,12 @@ VX_API_ENTRY vx_user_data_object VX_API_CALL vxCreateUserDataObject(
                 }
                 else
                 {
+                    vx_status status;
```

Comment:
It looks like I get a build error here now saying that status may not be initialized before it is checked on line 230.  Can this be initialized to VX_SUCCESS?

Reply:
done